### ISSUE TITLE
[Feature request]: Enforce micro break within long break

### ISSUE THREAD

**Issue by xeruf** [hovancik/stretchly#1363]
### Version

- [X] I'm using version 1.14.1

### Existing requests

- [X] I've checked Existing requests

Related to https://github.com/hovancik/stretchly/issues/775

### Advanced Preferences

- [X] I've checked Advanced Preferences

### What operating system are you using?

Windows

### Operating System Version

Arch

### Problem description

I find myself habitually clicking the postpone/skip button on long breaks - I have already disabled that option for mini breaks, but for long breaks I do sometimes need the ability to postpone - otherwise I resort to killing stretchly from the tty, which regularly sends me into turmoil due to missing breaks throughout the following hours.

### Possible solution

I want the option to be able to always postpone the long break, but only after a delay according to the micro break length, and never skip it - so that I have to intentionally use the tray menu if needed.
Maybe this setting would even invert the `breakPostponableDurationPercent`

### Alternative solutions

Just realized I can already raise the `breakPostponesLimit` to keep being reminded of the pending long break and enable strict mode in order to not habitually skip it. But this does not enforce a minimum break duration yet.
Alternatively this setting would replace the postpone button on the long break with one that converts it to a mini break.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
